node/src/components/block_proposer.rs

```diff
@@ -187,6 +188,8 @@ where
 struct BlockProposerReady {
     /// Set of deploys currently stored in the block proposer.
     sets: BlockProposerDeploySets,
+    /// "Unhandled" deploys are deploys that the BlockProposer has not yet seen.
```

Comment:
So _unhandled_ deploys are treated like finalized ones when picking deploys for the next block? Maybe it's worth adding a comment.

Reply:
Thanks, that could use a better explanation.